client/src/main/java/io/pravega/keycloak/client/KeycloakAuthzClient.java

```diff
@@ -56,12 +63,18 @@ public KeycloakAuthzClient(AuthzClient client, TokenCache tokenCache) {
         this.tokenCache = tokenCache;
     }
 
+    public KeycloakAuthzClient(AuthzClient client, TokenCache tokenCache, int httpMaxRetries, int httpInitialDelayMs) {
+        this(client, tokenCache);
+        this.httpMaxRetries = httpMaxRetries;
+        this.httpInitialDelayMs = httpInitialDelayMs;
+    }
+
     /**
      * Get an RPT for the target audience.
      *
      * @return a encoded RPT.
      */
-    public String getRPT() {
+    public String getRPT() {\
```

Comment:
there's debugging code I insert during testing that I remove at the last second before making the commit.  I actually left this random character by accident.  Fixing on my next commit.